fre/pp/configure_script_yaml.py

```diff
@@ -30,165 +31,94 @@ def yamlInfo(yamlfile,experiment,platform,target):
   t = target
   yml = yamlfile
 
+  # initialize rose suite config
+  rose_suite = metomi.rose.config.ConfigNode()
+  rose_suite.set(keys=['template variables', 'SITE'],              value='"ppan"')
+  rose_suite.set(keys=['template variables', 'CLEAN_WORK'],        value='True')
+  rose_suite.set(keys=['template variables', 'PTMP_DIR'],          value='"/xtmp/$USER/ptmp"')
+  rose_suite.set(keys=['template variables', 'DO_STATICS'],        value='True')
+  rose_suite.set(keys=['template variables', 'DO_TIMEAVGS'],       value='True')
+  rose_suite.set(keys=['template variables', 'DO_ATMOS_PLEVEL_MASKING'], value='True')
+  # disagreeable; these should be optional
+  rose_suite.set(keys=['template variables', 'DO_ANALYSIS_ONLY'],  value='False')
+  rose_suite.set(keys=['template variables', 'DO_MDTF'],  value='False')
+  rose_suite.set(keys=['template variables', 'PP_DEFAULT_XYINTERP'],  value='0,0')
```

Comment:
I'm not wild about setting these defaults that really shouldn't be required, but they're currently required.

Reply:
I guess we can live with these for now.. bigger fish to worry about